make cloning tutorial

joebeachjoebeach opened the pull request:
I did my best to keep the words "clone" and "cloning" out of the tutorial itself, but I'm having trouble coming up with a title for the project that doesn't use the word "clone."

Is it OK as it is, or should we think of a different title?

stevekrouse commented on woof/cloning-tutorial.html:
```diff
@@ -0,0 +1,123 @@
+				url: "./snippets/cloning-tutorial-step3.html"
+			},
+			{
+				text: "4) We need to make our sprites move. Many people try this code, but it won't work! Do you know why?",
```
When we tried this out yesterday, Una seemed confused if she was supposed to do something or nothing in this step. She said, "So am I supposed to copy and paste this code or make the sprites move on step 4?" Maybe add a note clarifying that you don't have to do anything at this step?

stevekrouse commented:
Another idea -- maybe think of a way to make it clear that Level 2 is all about cloning and that you should do this project first?

joebeachjoebeach commented:
I clarified the instructions in step 4, added a blurb at the beginning of the tutorial about how Level 2 is all about cloning, and I added "(Start Level 2 Here)" to the cloning tutorial link in index.html -- and since I did that, I figured I might as well add "(Start Here)" to Star Catcher.

Thoughts?

stevekrouse commented:
Looks great! I like having a "Start here" for star catcher and the cloning project. However, I think they can have the same exact text "Start here" and it can be implied that they mean "for this level". Also, I think it would look better to have "start here" on it's own line and a different color, potentially with a text outline or in green text. I want it to seem like the "right" thing to click. Potentially give it a badge or an icon...? Or you could but a vertical bar between it and the other projects of its level to demonstrate that it's the first one to do.

The name of the tutorial should probably be uniformly "Cloning Tutorial" instead of calling it "making copies of the same sprite" once you click on it.

I'd probably color the text green instead of yellow. I'd also give warning that the text cannot be copied at the top.

stevekrouse commented:
Also! Once we get this game made, I'd be excited about linking to it in the Woof documentation in the Control menu where we try to explain sprites. https://github.com/stevekrouse/WoofJS/blob/master/docs/index.html#L728

Potentially, we could say something along the lines of:

> There's no cloning in Woof, but you can create many sprites that do similar things in a loop:

```javascript
every(1, "second", () => {
  new Circle({
    x: randomX()
  })
})
```
> If you'd like to learn more, you can follow this tutorial...

joebeachjoebeach commented:
So, I did all three to test it out -- "Start Here" new color (green didn't look right, I felt yellow was better), star icon, and vertical bar. Let me know your thoughts and I can tweak accordingly.

Also added the warning and changed the highlight color from yellow to green.

That's a good idea about the documentation. Once this is up, I can create an issue in the Woof repository.
